src/plugins/substep/substep.js

```diff
@@ -118,6 +128,12 @@
             }
             var el = visible[ visible.length - 1 ];
             el.classList.remove( "substep-visible" );
+            if ( current > 0 &&
```

Comment:
Can you please add a comment above here to explain what this does. A short one is enough, but it's needed.